security/cmd/node_agent/na/nodeagent_test.go

```diff
@@ -47,6 +48,9 @@ func TestStartWithArgs(t *testing.T) {
 	}
 	signedCert := []byte(`TESTCERT`)
 	certChain := []byte(`CERTCHAIN`)
+	defer os.Remove(generalConfig.CAClientConfig.RootCertFile)
```

Comment:
Optional, put all os.Remove in as single defer func() {...}()?

Reply:
Sure.